test/ibmq/runtime/test_runtime.py

```diff
@@ -324,6 +322,22 @@ def test_program_params_namespace(self):
         params.param1 = "Hello World"
         self._run_program(program_id, inputs=params)
 
+    def test_program_upload_data_bytes(self):
+        """Test updating a program's string."""
+        program_id = self._upload_program()
+        self.runtime.update_program(program_id, 'print("Hello, Qiskit Provider.")'.encode())
+
+    @skipIf(os.name == 'nt', 'Test not supported on Windows')
```

Comment:
How do we support this for windows?

Reply:
We can! Described more [above](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1008#discussion_r677448632).